content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -14,201 +19,184 @@ the *bill control* policy chosen in the *Purchase* app settings.
-display on the :abbr:`RFQ (request for quotation)` form.
+(:dfn:`Reference Number`) can be changed from this pop-up window, if necessary.
+
+Once ready, click :guilabel:`Create Payment` to finish creating the :guilabel:`Vendor Bill`. Doing
+so displays a green :guilabel:`Paid` banner on the |RfQ| form.
 
-Create and manage vendor bills in Accounting
-============================================
+Manage vendor bills in Accounting
+=================================
 
 Vendor bills can also be created directly from the *Accounting* app, **without** having to create a
-purchase order first. To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`, and
-click :guilabel:`Create`. Doing so reveals a blank vendor bill detail form.
+purchase order first.
+
+Navigate to :menuselection:`Accounting app --> Vendors --> Bills`, and click :guilabel:`New`. Doing
+so reveals a blank vendor bill form.
+
+Add a vendor in the :guilabel:`Vendor` field. Then, under the :guilabel:`Invoice Lines` tab, click
+:guilabel:`Add a line` to add products.
 
-On this blank vendor bill detail form, add a vendor in the :guilabel:`Vendor` field, and add
-products to the :guilabel:`Product` lines (under the :guilabel:`Invoice Lines` tab), by clicking
-:guilabel:`Add a line`. Then, add a bill date in the :guilabel:`Bill Date` field, and any other
-necessary information. Finally, click :guilabel:`Confirm` to confirm the bill.
+Select a :guilabel:`Bill Date`, and configure any other necessary information. Finally, click
+:guilabel:`Confirm` to confirm the bill.
 
-From here, click the :guilabel:`Journal Items` tab to view (or change) the :guilabel:`Account`
-journals that were populated based on the configuration on the corresponding :guilabel:`Vendor` and
+Once confirmed, click the :guilabel:`Journal Items` tab to view the :guilabel:`Account` journals.
+These journals are populated based on the configuration on the corresponding :guilabel:`Vendor` and
 :guilabel:`Product` forms.
 
-Then, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note` to add credit or debit notes
-to the bill. Or, add a :guilabel:`Bill Reference` number (while in :guilabel:`Edit` mode).
+If necessary, click :guilabel:`Credit Note` to add a credit note to the bill. Additionally, a
+:guilabel:`Bill Reference` number can be added.
 
-Then, when ready, click :menuselection:`Register Payment --> Create Payment` to complete the
-:guilabel:`Vendor Bill`.
+Once ready, click :guilabel:`Register Payment`, followed by :guilabel:`Create Payment`, to complete
+the :guilabel:`Vendor Bill`.
 
 .. tip::
-   To tie the draft bill to an existing purchase order, click the drop-down menu next to
-   :guilabel:`Auto-Complete`, and select a :abbr:`PO (purchase order)` from the menu. The bill will
-   auto-populate with the information from the :abbr:`PO (purchase order)`.
+   To tie a draft bill to an existing purchase order, click the drop-down menu next to
```

Comment:
Totally optional rewording suggestion, since "tie" can translate incorrectly in other languages :

```suggestion
   To link a draft bill to an existing purchase order, click the drop-down menu next to
```